snapshot/lib/snapshot/runner.rb

```diff
@@ -83,35 +83,45 @@ def work
 
     def patch_scheme_to_save_screenshots
       scheme = Snapshot.config[:scheme]
-      scheme_absolute_path = File.join(File.expand_path(".", Snapshot.project.path), 'xcshareddata', 'xcschemes', scheme + ".xcscheme")
-
-      scheme_xml_doc = fetch_scheme_xml_file(scheme_file_path: scheme_absolute_path)
+      scheme_filename = scheme + ".xcscheme"
+      scheme_absolute_path = File.join(File.expand_path(".", Snapshot.project.path), 'xcshareddata', 'xcschemes', scheme_filename)
+      unless File.file?(scheme_absolute_path)
```

Comment:
One really cool way to do that in Ruby is to use wildcards

```ruby
path = Dir[File.join(File.expand_path(".", Snapshot.project.path), xcshareddata', 'xcschemes', scheme_filename].last

if path

else

end
```
This way you don't have that awkward `File.file?` but instead either have a path, or `nil`. This allows you to use wildcards also using `*`, so for example, if you don't care if it's called `xcshareddata`, or how many folders are inbetween
```ruby
# simplified
Dir[File.join(Snapshot.project.path, "**", scheme_file_name)
```